requirements.in

```diff
@@ -8,19 +8,20 @@ django-import-export
 django-haystack
 django-modeltranslation
 django-prettyjson
-django-recaptcha3
+django-recaptcha3 @ git+https://github.com/bartsanchez/django-recaptcha3.git@313509e40163ce381e7aa3decd1de1ed9f71d380
```

Comment:
It would be great to have a comment saying why this is and what has to happen to go back to the main one